### DISCUSSION TITLE
Create UI for turning Audit Log on/off

### DISCUSSION THREAD

**Discussion by benanhalt** [specify/specify7#5516] [Feature Requests]
The audit log tables can grow without limit. There should be some way to prune them.

**Comment by benanhalt** [comment]
By date? <-
By number of records?
Or just nuke everything?

**Comment by grantfitzsimmons** [comment]
https://github.com/specify/specify7/blob/2a259ca22b439ba71b9313705cb2a253ac8edd6e/specifyweb/frontend/js_src/lib/components/InitialContext/remotePrefs.ts#L218-L224

This is already supported in remote user prefs. 

As for configuring how long audit log entries are kept, I would need to ask @melton-jason to share the preference to add. We need to make sure it's actually happening when using Specify 7 alone as well.

**Comment by maxpatiiuk** [comment]
@grantfitzsimmons 

https://github.com/specify/specify7/blob/2a259ca22b439ba71b9313705cb2a253ac8edd6e/specifyweb/frontend/js_src/lib/components/InitialContext/remotePrefs.ts#L268-L275

Also keep in mind that @realVinayak will work on deprecating remote prefs and global prefs in sp7 when he has time, so the audit log system will be redone a bit. As far as UI, it will likely share a UI of user preferences
